dask/dataframe/tests/test_multi.py

```diff
@@ -1775,9 +1775,11 @@ def test_concat2():
         with warnings.catch_warnings(record=True) as w:
             expected = pd.concat(pdcase, sort=False)
 
-        ctx = FutureWarning if w else None
-
-        with pytest.warns(ctx):
+        if w:
+            with warnings.catch_warnings():
+                warnings.simplefilter("ignore", FutureWarning)
+                result = dd.concat(case)
+        else:
```

Comment:
This pattern feels awfully convoluted. What happens if you just remove all the warning handling?

Reply:
Yeah, agreed that if was added to handle some older version of `pandas` and is no longer needed, then we should remove it. Just noting that we have a `_check_warning` context manager

https://github.com/dask/dask/blob/9000abdd43772a82dcbf7999c5126b571d698d8a/dask/utils_test.py#L154-L164

for conditionally checking if a warning is raised and might be useful here

Reply:
Thank you for the suggestion!

> What happens if you just remove all the warning handling?

Good point! I tested with some older pandas versions locally, and the tests passed. I'll push a commit to check on our CI. :)

Reply:
Woo